lib/nerves/env.ex

```diff
@@ -386,6 +387,21 @@ defmodule Nerves.Env do
     end
   end
 
+  defp set_source_date_epoch() do
+    if source_date_epoch = source_date_epoch() do
+      System.put_env("SOURCE_DATE_EPOCH", source_date_epoch)
+    end
+  end
+
+  defp source_date_epoch() do
+    (System.get_env("SOURCE_DATE_EPOCH") || Application.get_env(:nerves, :source_date_epoch))
+    |> case do
+      nil -> nil
+      int when is_integer(int) -> Integer.to_string(int)
+      str when is_binary(str) -> str
```

Comment:
Could we have a check on the string to make sure that it's an integer? Specifically, I'd like to protect against someone setting `SOURCE_DATE_EPOCH` to the empty string and getting 1970 through out. I may have made that mistake.

Reply:
I updated this to validate the input